microsoft-edge/webview2/concepts/distribution.md

```diff
@@ -152,6 +152,32 @@ If you run the installer from an elevated process or command prompt, the Runtime
 
 Use the following online deployment workflow or offline deployment workflow to ensure that the Runtime is already installed before your app launches.  You can adjust your workflow depending on your scenario.  Sample code is available in the [Samples repo](https://github.com/MicrosoftEdge/WebView2Samples#webview2-deployment).
 
+#### Detect if a suitable WebView2 Runtime is already installed
+
+To verify that a WebView2 Runtime is installed, use one of the following approaches:
```

Comment:
```suggestion
To programmatically test whether a WebView2 Runtime is installed, use one of the following approaches.  This information is used as part of the deployment procedures.
```

Reply:
But only the second one is programmatically testing whether a WebView2 Runtime is installed, I think both approaches are covered as it is.